tests/installation/test_installer.py

```diff
@@ -102,17 +102,18 @@ def load(
 
 class Locker(BaseLocker):
     def __init__(self, lock_path: str | Path) -> None:
-        self._lock = TOMLFile(Path(lock_path).joinpath("poetry.lock"))
+        self._lock = Path(lock_path.joinpath("poetry.lock"))
         self._written_data = None
         self._locked = False
+        self._lock_data = None
```

Comment:
Reason: some tests were failing due to the lack of this property (that exists on parent `Locker`)